WordPress/Sniffs/WP/DeprecatedParameterValuesSniff.php

```diff
@@ -0,0 +1,216 @@
+					'alt'     => 'another settings group',
+					'version' => '3.5.0',
+				),
+			),
+		),
+		'bloginfo' => array(
+			1 => array(
+				'home' => array(
+					'alt'     => 'the "url" agrument',
+					'version' => '2.2.0',
+				),
+				'siteurl' => array(
+					'alt'     => 'the "url" agrument',
+					'version' => '2.2.0',
+				),
+				'text_direction' => array(
+					'alt'     => 'is_rtl()',
+					'version' => '2.2.0',
+				),
+			),
+		),
+		'get_bloginfo' => array(
+			1 => array(
+				'home' => array(
+					'alt'     => 'the "url" agrument',
+					'version' => '2.2.0',
+				),
+				'siteurl' => array(
+					'alt'     => 'the "url" agrument',
+					'version' => '2.2.0',
+				),
+				'text_direction' => array(
+					'alt'     => 'is_rtl()',
+					'version' => '2.2.0',
+				),
+			),
+		),
+		'register_setting' => array(
+			1 => array(
+				'misc' => array(
+					'alt'     => 'another settings group',
+					'version' => '3.0.0',
+				),
+				'privacy' => array(
+					'alt'     => 'another settings group',
+					'version' => '3.5.0',
+				),
+			),
+		),
+		'unregister_setting' => array(
+			1 => array(
+				'misc' => array(
+					'alt'     => 'another settings group',
+					'version' => '3.0.0',
+				),
+				'privacy' => array(
+					'alt'     => 'another settings group',
+					'version' => '3.5.0',
+				),
+			),
+		),
+	);
+
+	/**
+	 * Process the parameters of a matched function.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @param int    $stackPtr        The position of the current token in the stack.
+	 * @param array  $group_name      The name of the group which was matched.
+	 * @param string $matched_content The token content (function name) which was matched.
+	 * @param array  $parameters      Array with information about the parameters.
+	 *
+	 * @return void
+	 */
+	public function process_parameters( $stackPtr, $group_name, $matched_content, $parameters ) {
+		$this->get_wp_version_from_cl();
+		$param_count = \count( $parameters );
+		foreach ( $this->target_functions[ $matched_content ] as $position => $parameter_args ) {
+
+			// Stop if the position is higher then the total number of parameters.
+			if ( $position > $param_count ) {
+				break;
+			}
+
+			$this->process_parameter( $matched_content, $parameters[ $position ], $parameter_args );
+		}
+	}
+
+	/**
+	 * Process the parameter of a matched function.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @param string $matched_content The token content (function name) which was matched.
+	 * @param array  $parameter       Array with start and end token positon of the parameter.
+	 * @param array  $parameter_args  Array with alternative and WordPress deprecation version of the parameter.
+	 *
+	 * @return void
+	 */
+	protected function process_parameter( $matched_content, $parameter, $parameter_args ) {
+
+		$parameter_position = $this->phpcsFile->findNext(
+			Tokens::$emptyTokens,
+			$parameter['start'],
+			$parameter['end'] + 1,
+			true
+		);
+
+		if ( ! $parameter_position ) {
```

Comment:
Please check against `false`. While not problematic in this particular case, the `findPrevious()` sister-method can also return `0`, so strict checking is a good habit to get into for these things.